src/modules/Bootloader.rb

```diff
@@ -422,7 +422,23 @@ def ReadOrProposeIfNeeded
         Propose()
       else
         progress_orig = Progress.set(false)
+        if Stage.initial && Mode.update
+          # SCR has been currently set to inst-sys. So we have
+          # set the SCR to installed system in order to read
+          # grub settings
+          old_SCR = WFM.SCRGetDefault
+          new_SCR = WFM.SCROpen("chroot=#{Yast::Installation.destdir}:scr",
```

Comment:
I know that it probably gets loaded by some other module, but it is not imported. So please add import of "Installation".